react/Infos/styles.styl

```diff
@@ -1,9 +1,16 @@
 @require 'tools/mixins'
+@require 'settings/spaces'
```

Comment:
It looks like you wanted to use `spacing_values`, but you didn't. Was it your intention?

Reply:
Yes, good catch, thanks.